js/init.js

```diff
@@ -22,6 +22,6 @@
 
 /** global: OCA */
 
-$(document).ready(function() {
+window.addEventListener('DOMContentLoaded', function() {
```

Comment:
I think this now misses the case where the content has already loaded

> Most browsers provide similar functionality in the form of a DOMContentLoaded event. However, jQuery's .ready() method differs in an important and useful way: If the DOM becomes ready and the browser fires DOMContentLoaded before the code calls .ready( handler ), the function handler will still be executed. In contrast, a DOMContentLoaded event listener added after the event fires is never executed.

https://api.jquery.com/ready/

Reply:
There is no way the content is already loaded before the script is executed, I don't think this is possible with Nextcloud

Reply:
Not for scripts that are loaded with the page, but anything that is loaded dynamically. So I guess in this case it's okay, but we have to be careful :pray:

Reply:
But this is what is recommended in https://github.com/nextcloud/server/issues/20953 🙈 and what we use in https://github.com/nextcloud/server/pull/21921

Reply:
It wasn't me

![Bildschirmfoto von 2020-08-11 15-43-15](https://user-images.githubusercontent.com/1374172/89904583-67967d00-dbe9-11ea-8c99-384cfbf022a0.png)

Reply:
> It wasn't me

😆 I also never said that it was you, but that we recommend it in general 😝

Reply:
yeah we should be a tad more careful with this recommendation and add a note about the potential pitfall

Reply:
Ah right, we should just make sure every script using this is loaded directly via javascript :)